src/button/Button.ts

```diff
@@ -75,7 +74,7 @@ export default class Button<P extends ButtonProperties = ButtonProperties> exten
 
 	render(): DNode {
 		let {
-			describedBy,
+			aria = {},
```

Comment:
Are passed in `aria` properties meant to override defaults? 
Should passed in `aria` not be merged with default and *then* passed to `formatAriaProperties` ?

ie. 

``` ts
render() {
	const defaultAria = {
		haspopup: popup ? 'true' : null,
		controls: popup ? popup.id : null
	}

	const { aria = {} } = this.properties

	...formatAriaProperties({...defaultAria, ...aria})
}
```

Reply:
just a thought.

Reply:
I was actually thinking passed-in aria should not override the in-widget stuff, but if you can think of a good use case for overriding, I'm open to changing it